journal.lic

```diff
@@ -10,11 +10,14 @@ class WriteJournal
   def initialize
     settings = get_settings
     @journal_noun = settings.journal_noun
+    UserVars.journal_last_use ||= Time.now - 86400
 
     write_journal
   end
 
   def write_journal
+    return if Time.now - UserVars.journal_last_use < 86400
+
```

Comment:
❓ Do you need to set the UserVar to Time.now here?

Reply:
whoops